src/olympia/abuse/templates/abuse/emails/CinderActionDisableAddon.txt

```diff
@@ -1,8 +1,12 @@
 {% extends "abuse/emails/base.txt" %}{% block content %}
 Your {{ type }} {{ name }} was manually reviewed by the Mozilla Add-ons team {% if is_third_party_initiated %}based on a report we received from a third party{% else %}in an assessment performed on our own initiative of content that was submitted to Mozilla Add-ons{% endif %}.
 
+{% if has_attachment %}
+An attachment was provided. {% if dev_url %}To respond or view the file, visit {{ dev_url }}.{% endif %}
+
+{% endif %}
```

Comment:
Rather than repeating this in each file, stick it at the bottom of `abuse/emails/includes/policies.txt`, for templates that include that.